### PR TITLE
[new] add canvas-nest.js @1.0.0 

### PR DESCRIPTION
**Pull request by hustcc**
Hi @Piicksarn

repo url: [https://github.com/aTool-org/canvas-nest.js](https://github.com/aTool-org/canvas-nest.js)
npm package url: [https://www.npmjs.com/package/canvas-nest.js](https://www.npmjs.com/package/canvas-nest.js)

` Watch 9 ` ` Star 149 ` ` Fork 61 `

Would you mind checking this PR for me? Thank you~ :-D

 - [x] Not found on cdnjs repo
 - [x] No already exist issue and PR
 - [x] Notable popularity : Watch 9, Star 149, Fork 61
 - [x] Project has public repository on famous online hosting platform (or been hosted on npm)
 - [x] Has valid tags for each versions (for git auto-update)
 - [x] Auto-update setup
 - [x] Auto-update target/source is valid.
 - [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/canvas-nest.js/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "html5",
+    "nest"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/aTool-org/canvas-nest.js.git",
+    "basePath": "",
+    "files": [
+      "canvas-nest.js",
+      "canvas-nest.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/canvas-nest.js/package.json]: `"canvas-nest*.js"` will be easier.

File: ajax/libs/canvas-nest.js/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "filename": "canvas-nest.min.js",
+  "version": "1.0.0",
+  "description": "A nest backgroud of website draw on canvas use javascript, do not depend on jQuery. Less then 2kb.",
+  "homepage": "http://www.atool.org/",
+  "keywords": [
+    "canvas",
+    "html5",
+    "nest"
+  ],
+  "autoupdate": {
```
Comment (PeterDaveHello) [inline ajax/libs/canvas-nest.js/package.json]: There is no valid git tag in its repository so auto-update depends on git will not work.

Comment (PeterDaveHello) [inline ajax/libs/canvas-nest.js/package.json]: You can try npm auto-update: https://github.com/cdnjs/cdnjs#enabling-npmrecommended-or-git-auto-update

**Comment by PeterDaveHello** [comment]
Using 2 commits to add this lib makes no sense, so squash/fixup them into only one commit.

**Comment by hustcc** [comment]
thx, after done, give a new pr.

**Comment by PeterDaveHello** [comment]
@hustcc you can just upadte this PR by force push.

**Pull request closed without merging**